api/PeriodicSyncEvent.json

```
@@ -0,0 +1,146 @@
+{
+  "api": {
+    "PeriodicSyncEvent": {
+      "__compat": {
+        "support": {
+          "chrome": {
+            "version_added": "80"
+          },
+          "chrome_android": {
+            "version_added": "80"
+          },
+          "edge": {
+            "version_added": "80"
+          },
+          "firefox": {
+            "version_added": false
+          },
+          "firefox_android": {
+            "version_added": false
+          },
+          "ie": {
+            "version_added": false
+          },
+          "opera": {
+            "version_added": "67"
+          },
+          "opera_android": {
+            "version_added": "57"
+          },
+          "safari": {
+            "version_added": false
+          },
+          "safari_ios": {
+            "version_added": false
+          },
+          "samsunginternet_android": {
+            "version_added": false
+          },
+          "webview_android": {
+            "version_added": false
+          }
+        },
+        "status": {
+          "experimental": false,
```

If this is single-engine, then let's set this to `true` (here and throughout the PR)

```suggestion
          "experimental": true,
```

---

Oops, forgot to update those parts, thanks!